README.md

```diff
@@ -116,7 +116,7 @@ $ vendor/bin/bref local
 ## Deletion
 
 ```shell
-$ serverless remove
+$ bref remove
```

Comment:
Could you use `vendor/bin/bref` to be consistent with the rest of the documentation?